pkgs/top-level/all-packages.nix

```
@@ -29702,4 +29702,6 @@ in
   lc3tools = callPackage ../development/tools/lc3tools {};
 
   zktree = callPackage ../applications/misc/zktree {};
+
+  zydis = callPackage ../development/libraries/zydis {};
```

Please add it into a fitting category.

```suggestion
  zydis = callPackage ../development/libraries/zydis { };
```

---

Thanks, I moved this to `development/tools` instead.